packages/components/pie-cookie-banner/src/index.ts

```diff
@@ -10,7 +10,30 @@ const componentSelector = 'pie-cookie-banner';
 
 export class PieCookieBanner extends LitElement implements CookieBannerProps {
     render () {
-        return html`<h1 data-test-id="pie-cookie-banner">Hello world!</h1>`;
+        return html`
+        <aside data-test-id="pie-cookie-banner" class="c-cookieBanner">
+            <p class="c-cookieBanner-title">Cookies</p>
+            <div class="c-cookieBanner-body">
+                <p>We use our own and third party cookies and other tech to enhance and personalise your user experience,
+                optimize analytics, and show ads with third parties
+                <pie-link variant="inverse">(read our Statement)</pie-link>.
+                Necessary cookies are always set. Click <pie-link variant="inverse">Necessary only</pie-link>
+                to continue without accepting more. Click <pie-link variant="inverse">Manage preferences</pie-link>
+                to share your preferences or <pie-link variant="inverse">Accept all</pie-link>.</p>
+            </div>
+
+            <div class="c-cookieBanner-actions">
+                <pie-button variant="ghost-inverse" isFullWidth="true" size="small-expressive">
+                    Manage preferences
+                </pie-button>
```

Comment:
this will be a pie-link component, 
happy to leave it as is for now since the component is ready but lets remind each other later :p

Reply:
@raoufswe please see the notes in the PR description :)